### PR TITLE
Updated project

### PR DESCRIPTION
**Pull request by HytrexOfficial**
PR for updating the stopwatch to the newest packages.

Updated packages
migrated from packages.config to package reference
corrected the update url

### CODE REVIEW

**Comment by norwd** [comment]
Thanks for your PR @HytrexOfficial!

I've enabled the build workflow for your account but it looks like some of these changes aren't compiling. Can you please resolve these?

```
Run dotnet build --configuration Release --no-restore StopWatch.sln
  
C:\Program Files\dotnet\sdk\8.0.400\Microsoft.Common.CurrentVersion.targets(3413,5): error MSB3822: Non-string resources require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references. [D:\a\jirastopwatch\jirastopwatch\source\StopWatch\StopWatch.csproj]
C:\Program Files\dotnet\sdk\[8](https://github.com/jirastopwatch/jirastopwatch/actions/runs/10801492826/job/29963731694?pr=52#step:6:9).0.400\Microsoft.Common.CurrentVersion.targets(3413,5): error MSB3822: Non-string resources require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references. [D:\a\jirastopwatch\jirastopwatch\source\StopWatch\StopWatch.csproj]
Build FAILED.
C:\Program Files\dotnet\sdk\8.0.400\Microsoft.Common.CurrentVersion.targets(3413,5): error MSB3822: Non-string resources require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references. [D:\a\jirastopwatch\jirastopwatch\source\StopWatch\StopWatch.csproj]
C:\Program Files\dotnet\sdk\8.0.400\Microsoft.Common.CurrentVersion.targets(3413,5): error MSB3822: Non-string resources require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references. [D:\a\jirastopwatch\jirastopwatch\source\StopWatch\StopWatch.csproj]
    0 Warning(s)
    2 Error(s)
Time Elapsed 00:00:02.34
Error: Process completed with exit code 1.
```

**Comment by HytrexOfficial** [comment]
I will push the fix in the upcoming days. Seems that the migration of the nuget references caused some problems. I rolled it back so that it can build, but now some test are failing due to new restrictions of moq. 

Additionally i see that i need to sign the commits.

File: source/StopWatch/UI/MainForm.cs [hunk 1]
```diff
@@ -676,7 +676,7 @@ private void CheckForUpdates()
                                 latestRelease.TagName,
                                 currentVersion);
                             if (MessageBox.Show(msg, "New version available", MessageBoxButtons.YesNo) == DialogResult.Yes)
-                                System.Diagnostics.Process.Start("https://github.com/carstengehling/jirastopwatch/releases/latest");
+                                System.Diagnostics.Process.Start("https://github.com/jirastopwatch/jirastopwatch/releases/latest");
```
Comment (norwd) [inline source/StopWatch/UI/MainForm.cs:679]: Good Catch!

**Review by norwd — APPROVED** [review]

**Comment by norwd** [comment]
Thank you @HytrexOfficial, I've remove the signed commits requirement and that let the PR get merged.

**Comment by HytrexOfficial** [comment]
Oh, yeah thanks for the reminder 😅 the next pr will be with signed commits

**Comment by norwd** [comment]
> the next pr will be with signed commits

Don't worry about it :) I've just turned that requirement off, I don't think it's necessary and not everybody knows how to set them up

**Pull request merged**